src/com/phonebookservice/model/SocialProfile.java

```diff
@@ -0,0 +1,28 @@
+package com.phonebookservice.model;
+
+/**
+ * social profile class.
+ *
+ * @author Marwa Saleh
+ */
+public class SocialProfile {
+    public enum SocialMediaType {
```

Comment:
Should have OTHER as well

Reply:
You didn't answer this question !!

Reply:
why use configuration file when we have limited social profiles